src/pages/RepoPage/FlagsTab/subroute/FlagsTable/FlagsTable.tsx

```diff
@@ -27,10 +27,10 @@ import TableSparkline from './TableEntries/TableSparkline'
 import 'ui/Table/Table.css'
 
 interface FlagsTableHelper {
-  name: React.ReactNode
-  coverage: React.ReactNode
-  trend: React.ReactNode
-  delete: React.ReactNode
+  name: React.ReactElement
+  coverage: React.ReactElement
+  trend: React.ReactElement
+  delete: React.ReactElement | null
```

Comment:
nit: For delete this can stay React.ReactNode still right? since that includes null

Reply:
I played with this quite a bit because I was under that impression. It seems TypeScript says otherwise in this case 🤷